Sources/ClientRuntime/Telemetry/Logging/SDKLoggingSystem.swift

```diff
@@ -8,18 +8,18 @@
 import Logging
 
 public actor SDKLoggingSystem {
-    private static var isInitialized = false
-    private static var factories: [String: SDKLogHandlerFactory] = [:]
+    private var isInitialized = false
+    private var factories: [String: SDKLogHandlerFactory] = [:]
 
-    public static func add(logHandlerFactory: SDKLogHandlerFactory) {
+    public func add(logHandlerFactory: SDKLogHandlerFactory) {
         let label = logHandlerFactory.label
         factories[label] = logHandlerFactory
     }
 
-    public static func initialize(defaultLogLevel: SDKLogLevel = .info) async {
+    public func initialize(defaultLogLevel: SDKLogLevel = .info) async {
         if isInitialized { return } else { isInitialized = true }
         LoggingSystem.bootstrap { label in
-            if let factory = factories[label] {
+            if let factory = self.factories[label] {
```

Comment:
This is still throwing an actor-isolation warning (see below)

Reply:
(view the file to see the warning)

Reply:
Huh this is weird, when building locally I don't get this error. Regardless, will address it.